integration/fixtures/osio_config.toml

```diff
@@ -0,0 +1,41 @@
+logLevel = "ERROR"
+defaultEntryPoints = ["http"]
+[entryPoints]
+  [entryPoints.http]
+  address = ":8000"
+  [entryPoints.traefik]
+  address = ":7888"
+
+checkNewVersion = false
+
+[api]
+entryPoint = "traefik"
+
+[file]
+watch = false
+
+# rules
```

Comment:
This toml file is almost similar to one used in production.  It has all required combination of frontends and backends.